core/Pos/Core/Genesis/Types.hs

```diff
@@ -129,64 +128,46 @@ instance Buildable FakeAvvmOptions where
 
 -- | This data type contains various options presense of which depends
 -- on whether we want genesis for mainnet or testnet.
```

Comment:
Updating this comment may be a good idea too.

Reply:
Fixed.